scripts/generate_code.py

```diff
@@ -51,7 +51,7 @@
 # Get the location of the flatc executable, reading from the first command line
 # argument or defaulting to default names.
 flatc_exe = Path(
-    ("flatc" if not platform.system() == "Windows" else "flatc.exe")
+    ("flatc" if not platform.system() == "Windows" else "Release/flatc.exe")
```

Comment:
Actually, could you just remove the `Release/` part and put it in the build.yml as the `flatc` arg?